### PR TITLE
Moving changes from dev to staging

### PR DESCRIPTION
**Pull request by abdul-fs**
﻿## Summary
Pushing changes rom dev branch to staging branch.

Please note if fully resolves the issue per the acceptance criteria: Y/N Y

*Describe the pull request here, including any supplemental information needed to understand it.*
PR to mirror changes from dev to staging

## This pull request is ready to merge when...
- [x] Feature branch is starts with the issue number
- [ ] Is connected to its original issue via zenhub 👇
- [x] Tests have been updated 
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Sonarqube)
- [x] Server actions captured by logs (manual)
- [x] Documentation / readme.md updated (manual)
- [x] API docs updated if need (manual)
- [x] JSDocs updated (manual)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: server/dba/seeders/forests.js
Comment (mgwalker) [inline server/dba/seeders/forests.js:4]: Would the FIPS codes eventually go in this seeder file?

File: server/src/email/email-util.es6
Comment (mgwalker) [inline server/src/email/email-util.es6:123]: 🟠 

```suggestion
      console.log('Firewood permit successfully sent');
```

File: server/src/services/firewood-permit-service.es6
Comment (mgwalker) [inline server/src/services/firewood-permit-service.es6:164]: Is the HTML being converted to a PDF somewhere along the line? Not sure if that's something Nodemailer knows how to do, or if it's implemented elsewhere in here and I haven't seen it.

In any case, just wanted to clarify because I see three possibilities:
1. this is a placeholder until PDF creation is ready
2. PDF creation is happening somewhere and I didn't see it
3. this is a bug

I figure it's one of the first two (I see an empty `pdf.service.es6`, for example), but... it wouldn't be a good review if I didn't point and ask. 😛

**Review by mgwalker — APPROVED** [review]
Looks good to me. Just a couple of things to fix (marked with 🟠  &nbsp;in the comments below this post), and a couple of things that I wasn't sure if they needed to be fixed (marked with 🟡 ):
- 🟠  &nbsp;[number-of-cords.component.html](https://github.com/USDAForestService/fs-open-forest-platform/pull/1449/files/21586d7e6246698be8f5a7d3ac7f6dd3973c06fd#diff-1ea6a2dbcee95525d37c6cefcd4922e4) - label has the wrong ID, so it gets associated with the wrong form element. Accessibility
- 🟠  &nbsp;[email-util.es6](https://github.com/USDAForestService/fs-open-forest-platform/pull/1449/files/21586d7e6246698be8f5a7d3ac7f6dd3973c06fd#diff-86aceaeba96ec079923c1febfad49194) - the console log is about Snyk; having the right context there could be useful if this ever needs to be debugged!
- 🟡  &nbsp;[firewood-guidelines.component.html](https://github.com/USDAForestService/fs-open-forest-platform/pull/1449/files/21586d7e6246698be8f5a7d3ac7f6dd3973c06fd#diff-df8fe1be204bf18f5243912d43906c9f)
- 🟡  &nbsp;[order-confirmation.component.ts](https://github.com/USDAForestService/fs-open-forest-platform/pull/1449/files/21586d7e6246698be8f5a7d3ac7f6dd3973c06fd#diff-a1dab0722cd38aa452fcf7c6f702c645)

Other than that, I had some questions just related to normal Angular practices and some possible future optimizations/refactoring. Please don't feel any pressure to answer any of them since I know you're busy, but I love to learn new stuff and I've been away from Angular so long that... there's lots to learn. 😛 

I'm going to approve this so you don't have to wait for me to look at it again before you can merge it.

**Comment by abdul-fs** [comment]
@mgwalker @mtlaney are we good to push these changes in staging?

**Comment by mgwalker** [comment]
Good from my perspective

**Comment by mgwalker** [comment]
Can someone explain the process for pushing to staging? I thought this would have been merged last week which is why I reviewed it, but it's still open and new code is still being pulled into it. Is this supposed to happen on a regular cadence, or is it ad hoc? What's the criteria for pushing the merge button? Do design and product have insight into or participation in those decisions?

**Comment by abdul-fs** [comment]
> Can someone explain the process for pushing to staging? I thought this would have been merged last week which is why I reviewed it, but it's still open and new code is still being pulled into it. Is this supposed to happen on a regular cadence, or is it ad hoc? What's the criteria for pushing the merge button? Do design and product have insight into or participation in those decisions?



> Can someone explain the process for pushing to staging? I thought this would have been merged last week which is why I reviewed it, but it's still open and new code is still being pulled into it. Is this supposed to happen on a regular cadence, or is it ad hoc? What's the criteria for pushing the merge button? Do design and product have insight into or participation in those decisions?

PR is waiting with dev changes to push in staging. if mike and brian are good we push this in staging.

**Comment by mgwalker** [comment]
So the process is "Mike and Brian are ready to push to staging"?

**Comment by mtlaney** [comment]
The process is going to have to change going forward. Once @mgwalker is good we can move to staging. Brian and I are reviewing or approving everything to get to DEV. 

@abdul-fs once @mgwalker gives his blessing you are good to merge. If some time has passed between his review and the actual merge, we will either need to close the PR and open a new one, or re-request a review

**Comment by mgwalker** [comment]
To be clear, I didn't mean my comment to be a criticism. I'm genuinely curious when things get moved to staging. Is there something that triggers it? Is it just that enough stuff has been updated/fixed/added in dev? I was caught off-guard by an approved PR getting new commits and it made me realize I didn't actually know what the process was in the first place.

**Comment by mgwalker** [comment]
This PR has been open for a month and keeps growing. Do y'all need anything from me to merge it? I've looked through the changes since I clicked it off and it still looks good to me.

**Pull request merged**
